### DISCUSSION TITLE
Cruiser documentation

### DISCUSSION THREAD

**Discussion by JimmyS83** [devemux86/cruiser#219] [Q&A] [answered]
Hello,

I am struggle to find documentation to the Cruiser (original Kurviger app had a great one). Is it me, or its not done /published yet?

I would welcome it, as differences between font/display etc. are not so obvious and picture would be greatly help for getting the best from application.

Have a nice day!

**Comment by devemux86** [comment, marked as answer]
@JimmyS83 thanks for the question!

Cruiser is developed by 1 person and development already takes a lot of time to invest in documentation as well.
(Kurviger documentation was written by the community)

If you have any questions, please write so we can help you. :slightly_smiling_face: 

If there are settings that need explanation, we should probably improve their names / descriptions.

**Reply by JimmyS83** [reply]
Understand.. I dont have particulary "unclearness", I google a bit, and find this discussions, where some of interesting bits are discussed.. So in hope, it will not be burden, I just been curious.. I mean, it take a time to answer questions, so maybe side-notes, which could be later used for wiki base (I see you have wiki, brief on this moment).

Wish you good luck with the project, I found old Kurviger app usefull in certain situations, especially offline navi modes, so thanks for your work!

**Reply by devemux86** [reply]
@JimmyS83 thanks for the kind words!

Documentation is always on my mind, it's just that new features usually manage to take priority. :slightly_smiling_face: 

You can expect new great **offline** surprises in the next Cruiser updates.

**Reply by Cer-o** [reply]
Do you mean that you are solely responsible and that all the work is done by you?

**Reply by devemux86** [reply]
> Do you mean that you are solely responsible and that all the work is done by you?

:+1: 

All these years and before 2013, when Cruiser first appeared on Google Play!
Navigation algorithms, app features, all implemented by me. :slightly_smiling_face: 

I am also the developer of the [Mapsforge](https://github.com/mapsforge/mapsforge) and [VTM](https://github.com/mapsforge/vtm) map libraries we use in Cruiser.

**Comment by Ciutnik** [comment]
That's true. Through trial and error, I set the "Font size" and "Display size", which come in three different categories of settings. Of course, I'm very happy that these settings exist and find them very useful, but it can be confusing **for novice users**.

I understand that lack of time does not allow to create documentation, but a few words of description in small type for each option would be a huge help for beginners with Cruiser. Perhaps they could be introduced gradually, as the application develops and free time 🙂

**Reply by devemux86** [reply]
We have font / display size settings in 3 areas:
- **Map** settings for the map symbols and labels
- **Application | Appearance** for the screen panels
- **Application | Advanced** for the whole application

Indeed, we need to improve their names to help users understand what they are changing.

**Reply by Ciutnik** [reply]
Exactly - your description above is completely unambiguous and dispels doubts. Perhaps you could find one line under each setting that contains such a description? Probably the later it gets, the more difficult it will be 😉

**Comment by JimmyS83** [comment]
Maybe one thing come around today, Text-to-speech options: Normal / Advanced, I tried both, with test voice, it appeared quite same to me, and I think those options are not easilly explanatable within clean UI only imho... (ie. candidate for wiki)

**Reply by devemux86** [reply]
@JimmyS83 thanks for reporting it!

If there are more unexplained settings please mention them so we can improve them.

---

Advanced text-to-speech analyzes navigation instructions to deliver a better result.
For example, voice engines that could have problems speaking unit abbreviations.

e.g. Samsung TTS was reported to often not speak "100 m" as "100 meters".
Google TTS should not have such issue, both settings should work the same.

**Comment by devemux86** [comment]
In [Cruiser 3.0.25](https://play.google.com/store/apps/details?id=gr.talent.cruiser) several settings have new descriptions.

If you still find any settings that need improvements in their descriptions, please let me know.
